docs/myrocks-limitations.md

```diff
@@ -1,13 +1,61 @@
 # MyRocks limitations
 
-The MyRocks storage engine lacks the following features compared to InnoDB:
+## Online DDL limitations
 
-* [Online DDL](https://dev.mysql.com/doc/refman/{{vers}}/en/innodb-online-ddl.html) is not supported due to the lack of atomic DDL support.
+MyRocks has limited support for [Online DDL operations](https://dev.mysql.com/doc/refman/{{vers}}/en/innodb-online-ddl.html) due to the lack of atomic DDL. As a result the schema changes are more restricted compared to InnoDB.
 
-    * There is no `ALTER TABLE ... ALGORITHM=INSTANT` functionality
+### Traditional MyRocks DDL behavior
 
-    * A partition management operation only supports the `COPY` algorithms, which rebuilds the partition table and moves the data based on the new `PARTITION ... VALUE` definition. In the case of `DROP PARTITION`, the data not moved to another partition is deleted.
+| Operation type      | Examples                                         | ALGORITHM                   |
+|---------------------|--------------------------------------------------|-----------------------------|
+| Index operations    | `ADD INDEX`, `DROP INDEX`, `RENAME INDEX`        | `INPLACE` (always)          |
+| Column changes      | `ADD COLUMN`, `DROP COLUMN`, `MODIFY COLUMN`     | `COPY` (full table rebuild) |
+| Metadata changes    | `RENAME TABLE`, some `RENAME COLUMN` operations  | May be `INSTANT`            |
 
+**Note:** MyRocks does not support atomic DDL. Even metadata-only operations may require a full table rebuild, depending on the nature of the change.
+
+### Partition management support
+
+MyRocks supports `INPLACE` partition management for certain operations:
+
+```sql
+ALTER TABLE t1 DROP PARTITION p1, ALGORITHM=INPLACE;
+ALTER TABLE t1 ADD PARTITION (PARTITION p2 VALUES LESS THAN (MAXVALUE)), ALGORITHM=INPLACE;
+```
+The aforementioned operations no longer require a full table rebuild. However, operations that modify partitioning schemes, such as changing `VALUES LESS THAN`, still fall back to the `COPY` algorithm.
+
+**Note:** Dropping a partition permanently deletes any data stored in it unless that data is reassigned to another partition.
+
+### Instant DDL support    
+
+MyRocks has limited support for Instant DDL, which is disabled by default and controlled via configuration variables.
+
+To enable specific types of instant operations, use the following configuration options:
+
+| Configuration variable | Enables Instant DDL for  | ALGORITHM required         |
```

Comment:
No reason to have a column for information that doesn't change - remove ALGORITHM required.